cadvisor.go

```diff
@@ -186,3 +192,29 @@ func installSignalHandler(containerManager manager.Manager) {
 		os.Exit(0)
 	}()
 }
+
+func createCollectorHttpClient(collectorCert, collectorKey string) http.Client {
+	//Enable accessing insecure endpoints. We should be able to access metrics from any endpoint
+	tlsConfig := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+
+	if collectorCert != "" {
```

Comment:
I'm not a TLS expert, but if a certificate is provided why do we want to skip verification?